src/classes/RecipeBox.js

```diff
@@ -1,8 +1,27 @@
+const { recipeData } = require("../data/recipes");
+
 class RecipeBox {
   constructor(recipeData) {
     this.recipeData = recipeData;
   }
-  
+  storeTag(theUserInput) {
+    let matchingRecipesTags = [];
+    recipeData.forEach((recipe) => {
+      if(recipe.tags.includes(theUserInput)) {
+        matchingRecipesTags.push(recipe)
+        return matchingRecipesTags;
```

Comment:
I think this return has to be outside of both the if statement and the .forEach method, because the forEach method doesn't return anything.